### DISCUSSION TITLE
Is Gephi available for commercial use within the company?

### DISCUSSION THREAD

**Discussion by choikyungrak** [gephi/gephi#2635] [Q&A] [answered]
Hello. I'm an employee of a company. I want to use Gephi, which was familiar to me since I was in college. But licensing officer of my company announced that I have to confirm about legal approval. 
I cannot find this out ; Is Gephi(Trial) available for commercial use within the company?
Please help me to solve legal issue and finish my report with this app without legal issue.

**Comment by dirtyharrycallahan** [comment, marked as answer]
The license and faq are clearly documented on the landing page. Have you or your company's attorney or supply chain reviewed that documentation? Does your company have a processes for reviewing EULA/licenses against the intended use of software?
See: https://gephi.org/legal/faq/
